subt/test_name_decoder.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+from subt.name_decoder import parse_robot_name, split_multi
+
+
+class NameDecoderTest(unittest.TestCase):
+    def test_parse_robot_name(self):
+        steps = parse_robot_name('A10F100L')
+        self.assertEqual(steps, [('wait', 10), ('enter', None), ('left', 100), ('home', 200)])
+
+    def test_both_sides(self):
+        steps = parse_robot_name('A10F100LF200R')
+        self.assertEqual(steps, [('wait', 10), ('enter', None), ('left', 100), ('home', 200), ('enter', None), ('right', 200), ('home', 400)])
+
+    def test_variable_side(self):
+        steps = parse_robot_name('A10F100L200R')
```

Comment:
The difference between this name and the one above is F-minor. Would you mind adding documentation of the syntax at some point? And we may need a tool to generate the names or to interpret then in a human-readable way to reduce mistakes. Neither of this needs to be in the current PR.

Reply:
I was about to suggest the same :wink: But in general it seems to be something like `([0-9]+(L|R|C))+` for the actions plus the wait at the beginning. Is that a correct observation?

Reply:
There some extra `F`s needed. Oh well. Maybe we could drop the Fs and end up with something like `[A-Z]([0-9]+(L|R|C|W))+` for the whole name? An example would be: A wait 10s, go left 200s, return home 600s for code `A10W200L`

Reply:
Ok, I still don't get it. If there is an `F` somewhere it means return home?

Reply:
Except when it is after the first int where it means wait and enter?

Reply:
yes, correct - F was separator (underscore was not allowed) and we needed to wait at the beginning. I was thinking about new code 'H' for "home". Note, that it still needs to be relatively compact - 20 or 24 characters is the limit. We may want to add params for max speed, height??